src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -145,6 +194,9 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
     kwargs = {"subject_id": ("MONDO:%",), "object_id": prefix_args}
     with open(str(Path(output.replace("lexical", "lexical-2"))), "w") as f:
         filter_file(input=str(Path(output)), output=f, **kwargs)
+    t1 = datetime.now()  # todo: temp
+    print('match-mondo-sources-all-lexical complete in seconds:', (t1 - t0).seconds)  # todo temp
```

Comment:
## Outputs
@matentzn @twhetzel I will be uploading [outputs here](https://docs.google.com/spreadsheets/d/1topWKAf4-biD_dbmies-P3KQB53MKlovxIpUw0EYfDI/edit#gid=1579419867).